source/connection/connect.txt

```diff
@@ -1,8 +1,8 @@
 .. _java-connect-to-mongodb:
 
```

Comment:
S: should we change the file name to reflect the new title (mongoclient.txt), and add another ref anchor (like .. _java-create-client)? I don't think these are necessary changes but it would help make things less confusing for writers